ci-operator/config/openshift-knative/serving/openshift-knative-serving-release-next__411.yaml

```diff
@@ -1,253 +1,72 @@
-base_images:
-  base:
-    name: "4.11"
-    namespace: ocp
-    tag: base
-binary_build_commands: make install
 build_root:
   project_image:
     dockerfile_path: openshift/ci-operator/build-image/Dockerfile
-canonical_go_repository: knative.dev/serving
 images:
 - dockerfile_path: openshift/ci-operator/knative-images/activator/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/activator
   to: knative-serving-activator
-- dockerfile_path: openshift/ci-operator/knative-images/autoscaler/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/autoscaler
-  to: knative-serving-autoscaler
 - dockerfile_path: openshift/ci-operator/knative-images/autoscaler-hpa/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/autoscaler-hpa
   to: knative-serving-autoscaler-hpa
+- dockerfile_path: openshift/ci-operator/knative-images/autoscaler/Dockerfile
+  to: knative-serving-autoscaler
 - dockerfile_path: openshift/ci-operator/knative-images/controller/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/controller
   to: knative-serving-controller
-- dockerfile_path: openshift/ci-operator/knative-images/domain-mapping/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/domain-mapping
-  to: knative-serving-domain-mapping
 - dockerfile_path: openshift/ci-operator/knative-images/domain-mapping-webhook/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/domain-mapping-webhook
   to: knative-serving-domain-mapping-webhook
+- dockerfile_path: openshift/ci-operator/knative-images/domain-mapping/Dockerfile
+  to: knative-serving-domain-mapping
 - dockerfile_path: openshift/ci-operator/knative-images/migrate/Dockerfile
-  from: base
-  inputs:
-    bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/migrate
-  to: knative-serving-storage-version-migration
+  to: knative-serving-migrate
```

Comment:
Renamed. Can we live with that? Do we know all places where we need to change it?